venues/ICLR.cc/2019/Conference/python/invitations.py

```diff
@@ -246,21 +246,34 @@
 
 current_timestamp = lambda: int(round(time.time() * 1000))
 
-def enable_invitation(template_key, paper):
-    new_invitation = openreview.Invitation.from_json(
-        openreview.tools.fill_template(invitation_templates[template_key], paper))
+def enable_invitation(template_key, target=None):
```

Comment:
What does target mean?

Reply:
I want the invitations script to be able to modify templated invitations (per-paper, like official review) *or* non-templated invitations (like add bid). the "target" is the paper to template against. if it's None, it means that there is nothing to template against.

if you don't like the variable name, I don't mind changing it. but that's its purpose.